containers/src/Kokkos_DynamicView.hpp

```diff
@@ -373,9 +371,7 @@ class DynamicView : public Kokkos::ViewTraits<DataType, P...> {
     }
 
     void execute(bool arg_destroy) {
-      typedef Kokkos::RangePolicy<typename HostSpace::execution_space> Range;
-      // typedef Kokkos::RangePolicy< typename Impl::ChunkArraySpace< typename
-      // traits::memory_space >::memory_space::execution_space > Range ;
+      using Range = Kokkos::RangePolicy<typename HostSpace::execution_space>;
```

Comment:
I suppose this is fair game for a commented line that does not say why we should keep it